### ISSUE TITLE
Feature Request: Connect the UI with Python Chatbot over websocket

### ISSUE THREAD

**Issue by nikhil-chhabra** [dmitrizzle/chat-bubble#69]
Hi, I am looking to connect this to a websocket. I need the user inputs to be sent to the socket and the output from the socket to be given  back as bot's reply. Basically, I have a python model in place handling the conversations. I just want to connect this UI with the python model. Also, I need the bot to be reactive rather than initiative. Any help on this will be much appreciated. Thanks :)

**Comment by dmitrizzle** [comment]
I believe that if you get rid of `ice` object in your JSON, the bot will be reactive, i.e. it will not "greet" the user.

As for connecting it to a back-end, the way I did it is by loading generated JSON via AJAX and displaying the "typing" animation by invoking `think()` on the bot instance while that's happening. Once the data is received, I used `talk({ "response": { "says":[ apiResponseText ]} }, "response")` - basically creating a separate script file for each response. The second parameter in `talk` tells the app what is the default starting point key, in this case, it's "response".

It's been a while since I've implemented it that way but it's definitely doable. I can't share the code sample, unfortunately, since it was built under a private repo.

Let me know if I can help any further.
